getting-started/quickstart-guides/php.md

```diff
@@ -42,425 +42,1004 @@ It will prompt a few options. Select defaults by pressing enter.
+| height                        | h                       |  
+| width                         | w                       |  
+| aspectRatio                   | ar                      |  
+| quality                       | q                       |  
+| crop                          | c                       |  
+| cropMode                      | cm                      |  
+| x                             | x                       |  
+| y                             | y                       |  
+| focus                         | fo                      |  
+| format                        | f                       |  
+| radius                        | r                       |  
+| background                    | bg                      |  
+| border                        | bo                      |  
+| rotation                      | rt                      |  
+| blur                          | bl                      |  
+| named                         | n                       |  
+| overlayImage                  | oi                      |  
+| overlayX                      | ox                      |  
+| overlayY                      | oy                      |  
+| overlayFocus                  | ofo                     |  
+| overlayHeight                 | oh                      |  
+| overlayWidth                  | ow                      |  
+| overlayText                   | ot                      |  
+| overlayTextFontSize           | ots                     |  
+| overlayTextFontFamily         | otf                     |  
+| overlayTextColor              | otc                     |  
+| overlayAlpha                  | oa                      |  
+| overlayTextTypography         | ott                     |  
+| overlayBackground             | obg                     |  
+| overlayImageTrim              | oit                     |  
+| progressive                   | pr                      |  
+| lossless                      | lo                      |  
+| trim                          | t                       |  
+| metadata                      | md                      |  
+| colorProfile                  | cp                      |  
+| defaultImage                  | di                      |  
+| dpr                           | dpr                     |  
+| effectSharpen                 | e-sharpen               |  
+| effectUSM                     | e-usm                   |  
+| effectContrast                | e-contrast              |  
+| effectGray                    | e-grayscale             |  
+| original                      | orig                    |  
+| rotate                        | rt                      |  
+| overlayImageAspectRatio       | oiar                    |  
+| overlayImageBackground        | oibg                    |  
+| overlayImageBorder            | oib                     |  
+| overlayImageDPR               | oidpr                   |  
```

Comment:
This table is outdated, it doesn't have `raw`. Please ensure that both SDK, SDK's readme and quickstart guide mention and support all parameters same as Node.js latest version https://github.com/imagekit-developer/imagekit-nodejs/blob/master/README.md#list-of-supported-transformations
```php
| Supported Transformation Name | Translates to parameter |
|-------------------------------|-------------------------|
| height | h |
| width | w |
| aspectRatio | ar |
| quality | q |
| crop | c |
| cropMode | cm |
| x | x |
| y | y |
| focus | fo |
| format | f |
| radius | r |
| background | bg |
| border | b |
| rotation | rt |
| blur | bl |
| named | n |
| overlayX | ox |
| overlayY | oy |
| overlayFocus | ofo |
| overlayHeight | oh |
| overlayWidth | ow |
| overlayImage | oi |
| overlayImageTrim | oit |
| overlayImageAspectRatio | oiar |
| overlayImageBackground | oibg |
| overlayImageBorder | oib |
| overlayImageDPR | oidpr |
| overlayImageQuality | oiq |
| overlayImageCropping | oic |
| overlayImageFocus | oifo |
| overlayImageTrim | oit |
| overlayText | ot |
| overlayTextFontSize | ots |
| overlayTextFontFamily | otf |
| overlayTextColor | otc |
| overlayTextTransparency | oa |
| overlayAlpha | oa |
| overlayTextTypography | ott |
| overlayBackground | obg |
| overlayTextEncoded | ote |
| overlayTextWidth | otw |
| overlayTextBackground | otbg |
| overlayTextPadding | otp |
| overlayTextInnerAlignment | otia |
| overlayRadius | or |
| progressive | pr |
| lossless | lo |
| trim | t |
| metadata | md |
| colorProfile | cp |
| defaultImage | di |
| dpr | dpr |
| effectSharpen | e-sharpen |
| effectUSM | e-usm |
| effectContrast | e-contrast |
| effectGray | e-grayscale |
| original | orig |
| raw | `replaced by the parameter value` |
```